### PR TITLE
Improvement / HOLO 962 Clean lint warnings

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

Remove some lint warnings

**Before**
image

**After**
image

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] I have checked eslint

### CODE REVIEW

File: src/utils/network-monitor.ts [hunk 1]
```diff
@@ -1074,35 +1077,37 @@ export class NetworkMonitor {
         }
 
         if (
-          log.topics.length > 0 &&
-          log.topics[0] === event.sigHash &&
-          !this.isInterestingTransactionLogAlreadyIncluded(log, interestingTransactions)
+          log.topics.length === 0 ||
```
Comment (sogoiii) [inline src/utils/network-monitor.ts:1080]: @BravoNatalie i like the early exit strategy.

File: src/utils/network-monitor.ts [hunk 2]
```diff
@@ -1214,19 +1219,21 @@ export class NetworkMonitor {
...
-      if (isInBloom(block.logsBloom, filter.bloomValueHashed)) {
-        // check if there is additional validation required
-        if (filter.bloomFilterValidators) {
-          // iterate over each validator
-          for (const validator of filter.bloomFilterValidators) {
-            // if a match is found, then pass the transaction through
-            if (isInBloom(block.logsBloom, validator.bloomValueHashed)) {
-              return true
-            }
+      if (!isInBloom(block.logsBloom, filter.bloomValueHashed)) {
```
Comment (sogoiii) [inline src/utils/network-monitor.ts:1222]: @BravoNatalie i like the early exit strategy.

File: src/commands/analyze/index.ts [hunk 3]
```diff
@@ -334,6 +334,31 @@ export default class Analyze extends Command {
     ]
   }
 
+  isBeamingComplete(crossChainTx: CrossChainTransaction): boolean {
```
Comment (sogoiii) [inline src/commands/analyze/index.ts:337]: @BravoNatalie i like making these kinds of functions.

File: src/commands/analyze/index.ts [hunk 4]
```diff
@@ -499,104 +512,106 @@ export default class Analyze extends Command {
...
-        default:
-          operatorJobHash = decodeCrossChainMessageSentEvent(receipt, this.networkMonitor.operatorAddress)
-          if (operatorJobHash !== undefined) {
-            operatorJobPayload = decodeLzPacketEvent(receipt, this.networkMonitor.messagingModuleAddress)
+    let operatorJobHash: string | undefined
+    let operatorJobPayload: string | undefined
+    let args: any[] | undefined
+    let rawData: RawData | undefined
+
+    switch (this.environment) {
```
Comment (sogoiii) [inline src/commands/analyze/index.ts]: @BravoNatalie I would make this switch case a function. Maybe called `getJobHashAndPayload` ?

File: src/utils/utils.ts [hunk 5]
```diff
@@ -171,7 +171,7 @@ export const utf8ToBytes32 = (str: string): string => {
 }
 
 // turns multi-line query into single line and removes extra spaces
-export const cleanRequest = (query: any) => query.replace(/\n+ /g, '').replace(/\s+ /g, ' ')
+export const cleanRequest = (query: string): string => query.replace(/\n+ /g, '').replace(/\s+ /g, ' ')
```
Comment (ACC01ADE) [inline src/utils/utils.ts]: RegEx is very sensitive and the extra spaces in `/\n+ /g` are counted as well. Use this approach instead to remove all new lines, temporarily add a space, and then purge all spaces with more than two occurrences in a row -> `export const cleanRequest = (query: string): string => query.replace(/\n/g, ' ').replace(/\s{2,}/g, ' ')`

**Review by sogoiii — APPROVED** [review]
I think this is good. 👍

File: src/utils/network-monitor.ts [hunk 6]
```diff
@@ -1074,35 +1077,37 @@ export class NetworkMonitor {
...
-              interestingTransactions.push({
-                bloomId: 'TBD',
-                transaction: txMap[log.transactionHash],
-                log,
-                allLogs: allLogs[log.transactionHash]!,
-              } as InterestingTransaction)
-              tbdLogs.push(log.logIndex)
-            }
-          } else {
+          return
```
Comment (ACC01ADE) [inline src/utils/network-monitor.ts]: This should be `break` not `return`.

**Review by alexanderattar — APPROVED** [review]
This looks good to me. Thanks for making these improvements

**Review by sogoiii — APPROVED** [review]
👍

**Pull request merged**